frame/referenda/src/types.rs

```diff
@@ -143,7 +110,7 @@ pub struct Deposit<AccountId, Balance> {
 
 #[derive(Clone, Encode, TypeInfo)]
 pub struct TrackInfo<Balance, Moment> {
-	/// Name of this track. TODO was &'static str
+	/// Name of this track.
 	pub name: &'static str,
```

Comment:
I guess this means the name will stay as a `&'static str` type.

Reply:
Indeed.